crates/sdk/src/wallet/store.rs

```diff
@@ -534,11 +534,6 @@ impl Store {
         address: Address,
         force: bool,
     ) -> Option<Alias> {
-        // abort if the alias is reserved
-        if Alias::is_reserved(&alias).is_some() {
-            println!("The alias {} is reserved.", alias);
-            return None;
-        }
```

Comment:
@batconjurer is it ok to remove these lines? I don't see an issue doing so but want to make sure

Reply:
Yes, since we don't use aliases in genesis files anymore.